src/components/lib.js

```diff
@@ -1,15 +1,14 @@
-const titleCase = str => typeof str === 'string' ?
+export const titleCase = str => typeof str === 'string' ?
   str.charAt(0).toUpperCase() + str.slice(1).replace(/[^A-Z](?=[A-Z])/g, '$&\u00A0') :
   str;
 
 export function getModelMessage(i18n, key, defaultMessage) {
```

Comment:
Please, follow the previous review summary. **getMessage()** needs **payload** in certain cases, particularly, in case of Field/Instance Validation Errors => the function must have optional **payload** argument => utilize named arguments. Just as was stated in previous review summary.